views/homepage.handlebars

```diff
@@ -0,0 +1,22 @@
+<div>
+    <section class="hero">
+        <div class="large-6 columns">
+            <h1>Poke TeamWise</h1>
+        </div>
+        <div class="large-6 columns">
+            <h2>The place where Pokemon fans make battle-ready teams.</h2>
+        </div>
+    </section>
+    <section class="description">
+        <div class="large-6 columns">
+            <h2>Where Great Teams Are Made</h2>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet quisquam inventore quod velit eos consequuntur
+            neque tempore blanditiis atque officia! Reiciendis eum aliquid tempora fugiat et quam, asperiores sit
+            voluptatem?
+        </div>
+        <div class="large-6 columns">
+            <h1>Ready to ?</h1>
```

Comment:
Change text to "Ready to make a Pokemon team for the ages?" or something similar but catchy